src/ripple/core/impl/LoadEvent.cpp

```diff
@@ -29,73 +30,74 @@ LoadEvent::LoadEvent (
         LoadMonitor& monitor,
         std::string const& name,
         bool shouldStart)
-    : m_loadMonitor (monitor)
-    , m_isRunning (shouldStart)
-    , m_name (name)
-    , interval_ { std::chrono::steady_clock::now() }
+    : monitor_ (monitor)
+    , running_ (shouldStart)
+    , name_ (name)
+    , mark_ { std::chrono::steady_clock::now() }
     , timeWaiting_ {}
     , timeRunning_ {}
 {
 }
 
 LoadEvent::~LoadEvent ()
 {
-    if (m_isRunning)
+    if (running_)
         stop ();
 }
 
 std::string const& LoadEvent::name () const
 {
-    return m_name;
+    return name_;
 }
 
 std::chrono::milliseconds
-LoadEvent::getSecondsWaiting() const
+LoadEvent::waitTime() const
 {
-    return timeWaiting_;
+    // C++17: std::chrono::ceil
+    return ceil<std::chrono::milliseconds>(timeWaiting_);
```

Comment:
Yes; it is. Here's why I think this is fine: The existing code tracked time in seconds, using floating point, with millisecond precision. It exhibited this "weird" rounding you mention, where (in 1 place) 1ms became 0ms, and other values left unchanged.

We are tracking time using an integer in, at least, microsecond resolution. We are converting to milliseconds on demand, by rounding up. This means that if we spend 77 microseconds waiting and 20 microseconds running, we'll report 1 millisecond of runtime and 1 millisecond of wait time, and a total execution time of 2ms.

Given the intention of the code, as I understand it, and a conversation with @JoelKatz, this rounding up should be fine. David, any comments?

One change we could make would be to have the total execution time calculated in full precision (for our previous example: `77us + 20us` yielding `90us`) and _then_ using `ceil<std::chrono::millisecond>` on that result, to get `1ms`. In that case, the run & wait times might not add up to the execution time, but I don't think there a requirement for that to be the case.

What do you think?